cmake/std/atdm/ats2/environment.sh

```diff
@@ -0,0 +1,231 @@
+
+  if [[ "$ATDM_CONFIG_COMPILER" == *"CUDA"* ]]; then
+    sparc_tpl_ext=ats2-${arch}_cuda-10.1.243_gcc-7.3.1
+    sparc_tpl_mpi_ext=ats2-${arch}_cuda-10.1.243_gcc-7.3.1_spmpi-2019.06.24
+  else
+    sparc_tpl_ext=ats2-${arch}_gcc-7.3.1
+    sparc_tpl_mpi_ext=ats2-${arch}_gcc-7.3.1_spmpi-2019.06.24
+  fi
+elif [[ "$ATDM_CONFIG_COMPILER" == *"XL-2019.08.20_SPMPI-2019.06.24_DISABLED"* ]]; then
+  module load xl/2019.08.20
+  module load lapack/3.8.0-xl-2019.08.20
+  module load gmake/4.2.1
+
+  # Ninja not available for XL until cmake 3.17.0
+  export ATDM_CONFIG_USE_NINJA=OFF
+
+  # rabartl: ToDo: Above, we need to find a way to extract 'ats2-pwr9' out of
+  # this file for this to be general!
+
+  export CBLAS_ROOT=/usr/tcetmp/packages/lapack/lapack-3.8.0-P9-xl-2019.08.20
+  export LAPACK_ROOT=/usr/tcetmp/packages/lapack/lapack-3.8.0-P9-xl-2019.08.20
+  export COMPILER_ROOT=/usr/tce/packages/xl/xl-2019.08.20
+  export SPARC_HDF5=hdf5-1.8.20
+
+  # eharvey: TODO: remove COMPILER_ROOT and other exports below.
+  export PATH=${COMPILER_ROOT}/bin:${PATH}
+  export LD_LIBRARY_PATH=${COMPILER_ROOT}/lib:${LD_LIBRARY_PATH}
+  export BINUTILS_ROOT=/usr/tce/packages/gcc/gcc-7.3.1
+  export LIBRARY_PATH=${BINUTILS_ROOT}/lib
+  export LIBRARY_PATH=${CBLAS_ROOT}/lib:${LIBRARY_PATH}
+  export INCLUDE=${BINUTILS_ROOT}/include:${INCLUDE}
+  export CPATH=${BINUTILS_ROOT}/include:${CPATH}
+
+  if [[ "$ATDM_CONFIG_COMPILER" == *"CUDA"* ]]; then
+    export LD_LIBRARY_PATH=${BINUTILS_ROOT}/rh/lib/gcc/ppc64le-redhat-linux/7:${LD_LIBRARY_PATH}
+    sparc_tpl_ext=ats2-${arch}_cuda-10.1.243_xl-2019.08.20
+    sparc_tpl_mpi_ext=ats2-${arch}_cuda-10.1.243_xl-2019.08.20_spmpi-2019.06.24
+  else
+    sparc_tpl_ext=ats2-${arch}_xl-2019.08.20
+    sparc_tpl_mpi_ext=ats2-${arch}_xl-2019.08.20_spmpi-2019.06.24
+  fi
+else
+  echo
+  echo "***"
+  echo "*** ERROR: COMPILER=$ATDM_CONFIG_COMPILER is not supported on this system!"
+  echo "***"
+  return
+fi
+
+if [[ "$ATDM_CONFIG_COMPILER" == *"CUDA"* ]]; then
+  module load cuda/10.1.243
+
+  # OpenMPI Settings
+  export OMPI_CXX=${ATDM_CONFIG_NVCC_WRAPPER}
+  if [ ! -x "$OMPI_CXX" ]; then
+      echo "No nvcc_wrapper found"
+      return
+  fi
+  export OMPI_CC=`which gcc`
+  export OMPI_FC=`which gfortran`
+  export LLNL_USE_OMPI_VARS="y"
+
+  # CUDA Settings
+  export CUDA_LAUNCH_BLOCKING=1
+  export CUDA_MANAGED_FORCE_DEVICE_ALLOC=1
+  export TMPDIR=/tmp/$(whoami)
+
+  # ATDM Settings
+  export ATDM_CONFIG_USE_CUDA=ON
+  #export ATDM_CONFIG_Kokkos_ENABLE_Cuda_UVM=ON, handled by above export
+  export ATDM_CONFIG_USE_OPENMP=OFF
+  export ATDM_CONFIG_USE_PTHREADS=OFF
+  export ATDM_CONFIG_CUDA_RDC=OFF
+  export ATDM_CONFIG_CTEST_PARALLEL_LEVEL=4
+
+  # Kokkos Settings
+  export ATDM_CONFIG_Kokkos_ENABLE_Serial=ON
+  export ATDM_CONFIG_Kokkos_ENABLE_Cuda=ON
+  export ATDM_CONFIG_Kokkos_ENABLE_Cuda_Lambda=ON
+  export ATDM_CONFIG_Kokkos_ENABLE_Deprecated_Code=OFF
+  export KOKKOS_NUM_DEVICES=2
```

Comment:
@e10harvey, why is this not `KOKKOS_NUM_DEVICES=4`?

Reply:
No particular reason. Shall I change this to 4 for vortex?

Reply:
> No particular reason. Shall I change this to 4 for vortex?

Yes, I think 'vortex' nodes do have 4 GPUs per node, right?  That would allow more computing power to run the test suite.

Reply:
Correct.